frontend/src/components/TaskHeatmap.tsx

```diff
@@ -22,15 +22,15 @@ export const TaskHeatmap = () => {
     shallowEqual,
   );
 
-  const frequencies = matrix2d({ rows: 11, cols: 11 });
+  const frequencies = matrix2d({ rows: 5, cols: 5 });
   if (currentRoadmap) {
     currentRoadmap.tasks.map(ratingsSummaryByDimension).forEach((ratings) => {
       const value = ratings.get(TaskRatingDimension.BusinessValue);
       const work = ratings.get(TaskRatingDimension.RequiredWork);
       if (value && work) {
         const avgValue = Math.round(value.avg);
         const avgWork = Math.round(work.avg);
-        frequencies[10 - avgValue][avgWork] += 1;
+        frequencies[5 - avgValue][avgWork] += 1;
```

Comment:
> TaskHeatmap has a seemingly empty, transparent, row

```suggestion
        frequencies[5 - avgValue][avgWork - 1] += 1;
```

Previously this was off by one logically but the grid had some extra room. Now the work can also go out of bounds of the grid, which creates mostly empty column.